logos/benchmarks/benchmark_logos.py

```
@@ -1248,6 +1284,386 @@ def generate_charts(out_dir: Path, results: list[RequestResult], tracker, t0: fl
+    # -E: extended regex, no backslash escaping needed for () and |
+    # sudo is needed because sed -i writes a temp file in the same directory
+    sudo = "sudo " if use_sudo else ""
+    sed_expr = f"s/(^\\s*disable_sleep_mode:\\s*)(true|false)/\\1{new_val}/"
+    remote_cmd = f"{sudo}sed -E -i '{sed_expr}' {shlex.quote(config_file)}"
+    for host in hosts:
+        parts = ["ssh", "-o", "StrictHostKeyChecking=no"]
+        if ssh_key:
+            parts += ["-i", ssh_key]
+        parts += [f"{ssh_user}@{host}", remote_cmd]
+        print(f"  [logos] {host}: Set disable_sleep_mode={new_val} in {config_file}")
+        result = subprocess.run(parts)
```

Ruff (0.15.15)

[error] `subprocess` call: check for execution of untrusted input

(S603)